SofaKernel/modules/SofaBaseMechanics/DiagonalMass.h

```diff
@@ -227,7 +238,8 @@ class DiagonalMass : public core::behavior::Mass<DataTypes>
 
     virtual void reinit() override;
     virtual void init() override;
-
+    virtual void handleEvent(sofa::core::objectmodel::Event */*event*/) override;
```

Comment:
virtual is not needed (and may generate warning) if the function overrides another, the override keyword implies the virtual one. One should use:
```
void handleEvent(sofa::core::objectmodel::Event) override;
```
You can see [here](https://stackoverflow.com/questions/39932391/virtual-override-or-both-c) for more details